<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>renewPolicyOptions</class>
 <widget class="QDialog" name="renewPolicyOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Policy Options</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover{
	background-color: lightgrey;
color: black;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select at least 1 insurance policy to renew for this&lt;/p&gt;&lt;p&gt;new customer.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="carInsuranceOption">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Car Insurance</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="lifeInsuranceOption">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Life Insurance</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="homeInsuranceOption">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Home Insurance</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
